diff --git a/functions/misc_functions.lua b/functions/misc_functions.lua
index 32d878f..62a5f48 100644
--- a/functions/misc_functions.lua
+++ b/functions/misc_functions.lua
@@ -62,7 +62,7 @@ end
 
 function timer_checkpoint(label, type, reset)
   G.PREV_GARB = G.PREV_GARB or 0
-  if not G.F_ENABLE_PERF_OVERLAY then return end
+  if not (G.SETTINGS.perf_mode or G.SETTINGS.telemetry_log) then return end
   G.check = G.check or {
     draw = {
       checkpoint_list = {},
diff --git a/game.lua b/game.lua
index 2423983..8e0dd34 100644
--- a/game.lua
+++ b/game.lua
@@ -3262,8 +3262,9 @@ love.graphics.pop()
 
     timer_checkpoint('canvas', 'draw')
 
-    if not _RELEASE_MODE and G.DEBUG and not G.video_control and G.F_VERBOSE then 
+    if G.SETTINGS.perf_mode then 
         love.graphics.push()
+        love.graphics.origin()
         love.graphics.setColor(0, 1, 1,1)
         local fps = love.timer.getFPS( )
         love.graphics.print("Current FPS: "..fps, 10, 10)
