diff --git a/Mods/Cryptid/lib/calculate.lua b/Mods/Cryptid/lib/calculate.lua
index 7616b66..717401d 100644
--- a/Mods/Cryptid/lib/calculate.lua
+++ b/Mods/Cryptid/lib/calculate.lua
@@ -140,7 +140,7 @@ function Card:calculate_joker(context)
 		G.GAME.cry_double_scale = { double_scale = true } --doesn't really matter what's in here as long as there's something
 	end
 	--local orig_ability = copy_table(active_side.ability)
-	local in_context_scaling = false
+	-- CRY_DEAD_COPY_FIXED
 	local callback = context.callback
 	if active_side.ability.cry_possessed then
 		if
@@ -213,7 +213,6 @@ function Card:calculate_joker(context)
 	end
 	if not context.blueprint and (active_side.ability.set == "Joker") and not active_side.debuff then
 		if ret or trig then
-			in_context_scaling = true
 		end
 	end
 	return ret, trig
