diff --git a/functions/state_events.lua b/functions/state_events.lua
index 602eaa0..5a244d6 100644
--- a/functions/state_events.lua
+++ b/functions/state_events.lua
@@ -869,29 +869,38 @@ SMODS.displaying_scoring = true
                 end
             end
 
-            -- Calculate context.other_joker effects -- OTHER_KEY_HOISTED
+            -- Calculate context.other_joker effects -- OTHER_KEY_HOISTED CTX_TABLE_HOISTED
             local other_key = 'other_unknown'
             if _card.ability.set == 'Joker' then other_key = 'other_joker' end
             if _card.ability.consumeable then other_key = 'other_consumeable' end
             if _card.ability.set == 'Voucher' then other_key = 'other_voucher' end
+            local _other_ctx = {full_hand = G.play.cards, scoring_hand = scoring_hand, scoring_name = text, poker_hands = poker_hands}
+            _other_ctx[other_key] = _card
+            _other_ctx.other_main = _card
             for _, _area in ipairs(_joker_areas) do
                 for _, _joker in ipairs(_area.cards) do
                     -- TARGET: add context.other_something identifier to your cards
-                    local joker_eval,post = eval_card(_joker, {full_hand = G.play.cards, scoring_hand = scoring_hand, scoring_name = text, poker_hands = poker_hands, [other_key] = _card, other_main = _card })
+                    local joker_eval,post = eval_card(_joker, _other_ctx)
                     if next(joker_eval) then
                         if joker_eval.edition then joker_eval.edition = {} end
                         joker_eval.jokers.juice_card = _joker
                         table.insert(effects, joker_eval)
                         for _, v in ipairs(post) do effects[#effects+1] = v end
                         if joker_eval.retriggers then
+                            _other_ctx.retrigger_joker = true
+                            _other_ctx[other_key] = nil
+                            _other_ctx.other_main = nil
                             for rt = 1, #joker_eval.retriggers do
-                                local rt_eval, rt_post = eval_card(_joker, {full_hand = G.play.cards, scoring_hand = scoring_hand, scoring_name = text, poker_hands = poker_hands, [other_key] = _card, retrigger_joker = true})
+                                local rt_eval, rt_post = eval_card(_joker, _other_ctx)
                                 if next(rt_eval) then
                                     table.insert(effects, {retriggers = joker_eval.retriggers[rt]})
                                     table.insert(effects, rt_eval)
                                     for _, v in ipairs(rt_post) do effects[#effects+1] = v end
                                 end
                             end
+                            _other_ctx.retrigger_joker = nil
+                            _other_ctx[other_key] = _card
+                            _other_ctx.other_main = _card
                         end
                     end
                 end
